src/config/snowflake.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+return [
+    'machine_id' => env('SNOWFLAKE_MACHINE_ID', -1)
```

Comment:
Providing a legitimate default value like `0` or `1` wouldnt be great because if the developer forgets to set the env key, they're likely to start generating duplicate IDs across their various deployments, because the machine id part won't be unique any more. So by setting it to a default value that will throw an exception forces the developer to make a conscious effort to assign a unique value to each environment using their `.env` file.